### DISCUSSION TITLE
Does stcgal support reload and download when target file is modified?

### DISCUSSION THREAD

**Discussion by jmjoy** [grigorig/stcgal#98] [Q&A] [answered]
Just like this option in `stc-isp.exe`:
![image](https://github.com/grigorig/stcgal/assets/8677974/4850fa20-818d-4783-bac9-285bff95bcb5)

**Comment by grigorig** [comment, marked as answer]
This sound quite useful for some workflows, but currently isn't supported. Feel free to submit a PR if you want to implement it.
